.yarnrc.yml

```diff
@@ -44,13 +44,15 @@ plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
     spec: "@yarnpkg/plugin-interactive-tools"
   - ./.hooks/plugin-warning-logger.js
+  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
+    spec: "@yarnpkg/plugin-workspace-tools"
```

Comment:
This plugin provides `yarn workspaces foreach ...`.
https://yarnpkg.com/cli/workspaces/foreach